.github/workflows/bundlesize.yml

```diff
@@ -23,6 +23,7 @@ jobs:
               run: echo "CI_COMMIT_SHA=${{ github.event.pull_request.head.sha}}" >> $GITHUB_ENV
 
             - name: Run BundleMon
+              if: secrets.BUNDLEMON_PROJECT_APIKEY
```

Comment:
This can safely be removed. I upgrade bundlemon to a version that no longer requires this trick

Reply:
Removed this line just now.